### PR TITLE
Small code improvements (proposal)

### PR DESCRIPTION
**Pull request by Ronny25**
Tiny PR with some code improvements if you don't mind.

**NOTE**: should be merged after [PR](https://github.com/PRNDcompany/ReactSvgStarRating/pull/2)

### CODE REVIEW

File: src/StarRating.tsx [hunk 1]
```diff
@@ -103,33 +114,41 @@ const StarRating: React.FC<StarRatingProps> = ({
...
-      return isHalfSelectedHover ? DEFAULT_COLOR : index <= selectedStarIndex ? activeColor : DEFAULT_COLOR;
-    } else {
-      return isHalfSelectedHover ? activeColor : getLeftColor(index);
+      if (!isHalfSelectedHover && index > selectedRating) {
+        return activeColor;
+      }
+
+      return DEFAULT_COLORS.DEFAULT_COLOR;
     }
+
```
Comment (nick-prnd) [inline src/StarRating.tsx:132]:
```
if (isLeftSideHover) {
      if (!isHalfSelectedHover) {
        if (index > selectedRating)
          return activeColor;
        else
          return getLeftColor(index);
      }
      return DEFAULT_COLORS.DEFAULT_COLOR;
    }
```
I think we should return `getLeftColor(index)` in this condition `index <= selectedRating`.

**Review by nick-prnd — COMMENTED** [review]
Thank you very much for your help!!

File: src/StarRating.tsx (hunk 1 above)
Comment (Ronny25) [reply src/StarRating.tsx:132]: I think it's included.
Condition is just the opposite - `index > selectedRating` and in **else** `getLeftColor(index)` is returned.
But you can simply decline this PR 😉

Comment (nick-prnd) [reply src/StarRating.tsx:132]: I added `getLeftColor(index)` in the `else`.
Could you modify your PR like that?
I would like to merge your code.

Comment (Ronny25) [reply src/StarRating.tsx:132]: Sure, will double check that!
But will be available in 2 weeks if it's ok 😀

**Pull request closed without merging**